zerver/views/auth.py

```diff
@@ -96,15 +97,18 @@ def get_safe_redirect_to(url: str, redirect_host: str) -> str:
     else:
         return redirect_host
 
-def confirm_email_change(request: HttpRequest, confirmation_key: str) -> HttpResponse:
+def confirm_email_change(request: HttpRequest, confirmation_key: str,
+                         user_profile: UserProfile) -> HttpResponse:
```

Comment:
should we name this `acting_user` instead?

Reply:
Not sure. But better than `user_profile` as it could be confusing sometimes. So changed it.